### PR TITLE
CreatePickup component

### PR DESCRIPTION
**Pull request by Xbaal**
html code from mvp-design
not much done yet.

### CODE REVIEW

**Comment by D0nPiano** [comment]
Mostly working now.

@tiltec and @nicksellen - lint is failing, because the backend needs `max_collectors` field for pickup-date creation, which is not camelCased. Is there a workaround for lint, or should the backend be changed from `max_collectors` to `maxCollectors`?

**Comment by nicksellen** [comment]
I think the backend should stay as _snake_case_  as that is the convention in python. For now, you can use [eslint's disable](http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments) comments.

For later, maybe we can implement a function/service to automatically convert between snake_case and camelCase as needed (as it gets quite verbose).

**Comment by D0nPiano** [comment]
Ah, thanks! eslint disable is what i was looking for! :+1:

**Comment by D0nPiano** [comment]
- [x]  Added as Dialog
- [ ]  Test for createPickup() // currently failing because sql of `api/pickup-date -> date` converts to local time in some cases
- [x] update Pickuplist afterwards

**Comment by D0nPiano** [comment]
I can't really think of anything else at the moment. Is it done? (:

File: client/app/components/_createPickup/createPickup.spec.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            return 22;
+          }
+        },
+        maxCollectors: 5
+      };
+      controller.createPickup();
+
+      $httpBackend.expectPOST("/api/pickup-dates/", {
+        max_collectors: 5, // eslint-disable-line
+        date: controller.date.toISOString(),
```
Comment (tiltec) [inline client/app/components/_createPickup/createPickup.spec.js]: @D0nPiano - I changed it to use the `toISOString()` function. it works quite well for me, what do you think about it?

Comment (D0nPiano) [inline client/app/components/_createPickup/createPickup.spec.js]: Looks way better! Thanks for changing it. (and the slider looks awesome too :+1: )

**Comment by Xbaal** [comment]
I would like to see some error message, if the time isn't correct. I will have a look into this

**Comment by D0nPiano** [comment]
@Xbaal Maybe we could also add a tooltip (https://material.angularjs.org/latest/demo/tooltip) that explains the correct format.

**Comment by tiltec** [comment]
Looks good so far, I'll file an issue for smaller improvements.
Would merge.

**Pull request merged**